internal/pkg/supervisor/full.go

```diff
@@ -0,0 +1,52 @@
+// (c) Cartesi and individual authors (see AUTHORS)
+// SPDX-License-Identifier: Apache-2.0 (see LICENSE)
+
+package supervisor
```

Comment:
This sub-package is redundant. The cmd package has nothing else, so we should move all these files to `cmd/<binary-name>/`. Let's start simple. When we do so, we can use internal symbols for everything (symbols that start with a lowercase).